test/tests/constraints/equal_value_embeded_constraint/tests

```diff
@@ -0,0 +1,72 @@
+[Tests]
```

Comment:
It looks like a lot of these tests are essentially identical except that one or two options are changed. Try to just use one input file, and use `cli_args` to set the parameters for running the test with different options. If you do that, you also need to use `prereq` to make sure two tests don't run at the same time in parallel.

Also, use the newly added `requirement`, `design`, and `issues` options here. Look at modules/tensor_mechanics/test/tests/2D_different_planes/tests for an example.